code/modules/antagonists/eldritch_cult/knowledge/rust_lore.dm

```diff
@@ -108,17 +108,17 @@
 	desc = "You can now create eldritch armor using a table and a gas mask."
 	gain_text = "Rusted Hills have welcomed Blacksmith with their generosity."
 	cost = 1
-	next_knowledge = list(/datum/eldritch_knowledge/rust_regen,/datum/eldritch_knowledge/flesh_ghoul)
+	next_knowledge = list(/datum/eldritch_knowledge/rust_regen,/datum/eldritch_knowledge/cold_snap)
 	required_atoms = list(/obj/structure/table,/obj/item/clothing/mask/gas)
 	result_atoms = list(/obj/item/clothing/suit/hooded/cultrobes/eldritch)
 
 /datum/eldritch_knowledge/essence
 	name = "Priest's ritual"
-	desc = "You can now transmute a tank of water into a bottle of eldritch water."
+	desc = "You can now transmute a tank of water and a glass shard into a bottle of eldritch water."
```

Comment:
kind of feel like this should use some sort of reagent container, not a glass bottle

Reply:
doesnt really make a diffrence

Reply:
you're using glass shards in a lot of recipes already, though- this might add some more variety to the ritual components